### DISCUSSION TITLE
community members: do not post here!

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#84] [Admin General]
this category is for admins only. please go [here](https://github.com/mariocraft987/bark.github.io/discussions/categories/general) to post in "general."

**Comment by Singh-Prince-Rinku** [comment]
Response #1

**Reply by mariocraft987** [reply]
Ummm...

**Reply by raynepaws** [reply]
who are you??

**Reply by mariocraft987** [reply]
That's what I was thinking!

**Reply by rubiidev18alt** [reply]
he is testing comments

**Reply by mariocraft987** [reply]
hmmm.. how'd you get him?

**Reply by Singh-Prince-Rinku** [reply]
i was here for testing. chill guys

**Reply by raynepaws** [reply]
sorry lol

**Reply by raynepaws** [reply]
how'd you find this repository tho?

**Reply by Singh-Prince-Rinku** [reply]
from discussion community

**Reply by mariocraft987** [reply]
oh
